components/resources/decentralized_dns_strings.grdp

```diff
@@ -16,7 +16,7 @@
     Enable support of Ethereum Name Service (ENS) in Brave?
   </message>
   <message name="IDS_ENS_OPT_IN_PRIMARY_PARAGRAPH" desc="Main paragraph of ENS opt-in interstitial page.">
-    Brave will be using Infura to resolve .eth domain names that are on Ethereum Name Service (ENS). Brave hides your IP address. If you enable this, Infura will see that someone is trying to visit these .eth domains but nothing else. See Infura's <ph name="BEGIN_INFURA_TOU_LINK">&lt;a target="_blank" href="$1"&gt;</ph>terms of use<ph name="END_INFURA_TOU_LINK">&lt;/a&gt;</ph> and <ph name="BEGIN_INFURA_PRIVACY_LINK">&lt;a target="_blank" href="$2"&gt;</ph>privacy policy<ph name="END_INFURA_PRIVACY_LINK">&lt;/a&gt;</ph>.
+    Brave will be using <ph name="PROVIDER_NAME">$1<ex>Infura</ex></ph> to resolve <ph name="DOMAIN_LIST">$2<ex>.eth</ex></ph> domain names that are on Ethereum Name Service (ENS). Brave hides your IP address. If you enable this, <ph name="PROVIDER_NAME">$1<ex>Infura</ex></ph> will see that someone is trying to visit these <ph name="DOMAIN_LIST">$2<ex>.eth</ex></ph> domains but nothing else. See <ph name="PROVIDER_NAME">$1<ex>Infura</ex></ph>'s <ph name="BEGIN_PROVIDER_TOU_LINK">&lt;a target="_blank" href="$3"&gt;</ph>terms of use<ph name="END_PROVIDER_TOU_LINK">&lt;/a&gt;</ph> and <ph name="BEGIN_PROVIDER_PRIVACY_LINK">&lt;a target="_blank" href="$4"&gt;</ph>privacy policy<ph name="END_PROVIDER_PRIVACY_LINK">&lt;/a&gt;</ph>.
```

Comment:
Do you think this one and the previous one (IDS_UNSTOPPABLE_DOMAINS_OPT_IN_PRIMARY_PARAGRAPH) could be unified into one string by removing the second use of `<ph name="DOMAIN_LIST">$2<ex>.eth</ex></ph>` here and making `Unstoppable Domains`/`Ethereum Name Service (ENS)` a placeholder?